main/hgrep.hs

```diff
@@ -44,13 +45,13 @@ getAllHsFiles fp = do
   isDir <- D.doesDirectoryExist fp
   if isDir
   then do
-       fs <- L.map (fp FP.</>) <$> D.listDirectory fp
-       foldMap getAllHsFiles fs
+    fs <- L.map (fp FP.</>) <$> D.listDirectory fp
```

Comment:
It looks like you imported Data.List only for `map`, can you replace this with `fmap` and remove the import?

Reply:
Sure!